spec/scrabble_spec.rb

```diff
@@ -0,0 +1,72 @@
+require './lib/scrabble.rb'
+
+describe Scrabble do
+
+  context "Scoring words" do
+
+    it "returns Error if the word has too many letters" do
+      expect(Scrabble::Scrabble.score("watermelon")).to eq("Error")
+    end
+
+    it "returns Error if nothing is entered" do
+      expect(Scrabble::Scrabble.score("")).to eq("Error")
+    end
+
+    it "returns Error if the word has numbers in it" do
+      expect(Scrabble::Scrabble.score("app3le")).to eq("Error")
+    end
+
+    it "returns Error if the word contains special characters" do
+      expect(Scrabble::Scrabble.score("ap@le")).to eq("Error")
+    end
+
+    # it "returns 1 for a" do
+    #   expect(Scrabble::Scrabble.score("a")).to eq(1)
+    # end
+
+    # it "returns 4 for ap" do
+    #   expect(Scrabble::Scrabble.score("ab")).to eq(4)
+    # end
+
+    [
+      ["a",  1],
+      ["ab", 4],
+      ["apple", 9],
+      ["A", 1],
+      ["BAnaNA", 8]
+    ].each do |word, score|
+      it "returns #{score} for #{word}" do
+        expect(Scrabble::Scrabble.score(word)).to eq(score)
+      end
+    end
+  end
+
+  context "Highest scoring word" do
+
+    # it "returns an array of values" do
```

Comment:
Yes, I left them so I could  see what my thinking was while defining self.highest_score_from(array_of_words) if I wanted to rewrite some of it.